src/main/java/com/diffblue/cover/annotations/ContributionFromDiffblue.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2025 Diffblue Limited.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package com.diffblue.cover.annotations;
+
+/**
+ * Empty interface, required, with the Category annotation, to label JUnit 4 tests in the following
+ * manner: @Category(MaintainedByDiffblue.class).
```

Comment:
nit: this seems wrong, I assume it was copied over? 
```suggestion
 * manner: @Category(ContributionFromDiffblue.class).
```

Reply:
yep, well spotted